examples/EvaluationDataToApplicationInsights/README.md

```diff
@@ -0,0 +1,77 @@
+# Evaluation Data to Application Insights
+
+This sample shows how to send evaluation data to Application Insights. Evaluation data occurs each time a feature result is determined by the Feature Manager.
+
+## Quickstart
+
+To run this sample, follow these steps:
+
+1. Set the project as the startup project.
+2. Run the project.
+3. Check the Output tab for `Application Insights Telemetry`.
+
+Example Output:
+
+```
+Application Insights Telemetry: {"name":"AppEvents","time":"2023-11-07T19:14:54.3549353Z","tags":{"ai.application.ver":"1.0.0.0"},"data":{"baseType":"EventData","baseData":{"ver":2,"name":"Vote","properties":{"AspNetCoreEnvironment":"Development","DeveloperMode":"true"},"measurements":{"ImageRating":3}}}}
```

Comment:
Where can I find these outputs? I did not find the Output tab for "Application Insights Telemetry"

Reply:
I can only find them in the Debug model.

Reply:
In the output tab of visual studio.